content/integrations/integrating-npm-with-external-services/using-private-packages-in-a-ci-cd-workflow.mdx

```diff
@@ -19,50 +19,48 @@ If you use a different CI/CD provider, or if you need to install private package
 
 ## Create a new access token
 
-Create a new access token that will be used only to access npm packages from a CI/CD server.
+Create a new granular access token that will be used only to access npm packages from a CI/CD server.
```

Comment:
imho lets keep this as access token itself, in future we should drop the granular term for everywhere because there will be only one type of access token in future